src/test/java/com/commercetools/project/sync/customobject/CustomObjectSyncerTest.java

```diff
@@ -1,132 +1,143 @@
-  //                .plusPredicates(
-  //                    customObjectQueryModel ->
-  //                        customObjectQueryModel.container().isNotIn(excludedContainerNames)));
-  //    assertThat(customObjectSyncer.getSync()).isExactlyInstanceOf(CustomObjectSync.class);
-  //  }
-  //
-  //  @Test
-  //  void transform_ShouldConvertResourcesToDrafts() {
-  //    // preparation
-  //    final CustomObjectSyncer customObjectSyncer =
-  //        CustomObjectSyncer.of(
-  //            mock(SphereClient.class), mock(SphereClient.class), getMockedClock(), "", false);
-  //
-  //    final CustomObject<JsonNode> customObject1 = mock(CustomObject.class);
-  //    when(customObject1.getContainer()).thenReturn("testContainer1");
-  //    when(customObject1.getKey()).thenReturn("testKey1");
-  //    when(customObject1.getValue()).thenReturn(JsonNodeFactory.instance.textNode("testValue1"));
-  //
-  //    final CustomObject<JsonNode> customObject2 = mock(CustomObject.class);
-  //    when(customObject2.getContainer()).thenReturn("testContainer2");
-  //    when(customObject2.getKey()).thenReturn("testKey2");
-  //    when(customObject2.getValue()).thenReturn(JsonNodeFactory.instance.booleanNode(true));
-  //    final List<CustomObject<JsonNode>> customObjects = asList(customObject1, customObject2);
-  //
-  //    // test
-  //    final CompletionStage<List<CustomObjectDraft<JsonNode>>> draftsFromPageStage =
-  //        customObjectSyncer.transform(customObjects);
-  //
-  //    // assertions
-  //    assertThat(draftsFromPageStage)
-  //        .isCompletedWithValue(
-  //            customObjects.stream()
-  //                .map(
-  //                    customObject ->
-  //                        CustomObjectDraft.ofUnversionedUpsert(
-  //                            customObject.getContainer(),
-  //                            customObject.getKey(),
-  //                            customObject.getValue()))
-  //                .collect(toList()));
-  //  }
-  //
-  //  @Test
-  //  void getQuery_doNotSyncProjectSyncCustomObjects_ShouldBuildCustomObjectQuery() {
-  //    // preparation
-  //    final String runnerName = "testRunnerName";
-  //    final CustomObjectSyncer customObjectSyncer =
-  //        CustomObjectSyncer.of(
-  //            mock(SphereClient.class),
-  //            mock(SphereClient.class),
-  //            getMockedClock(),
-  //            runnerName,
-  //            false);
-  //
-  //    // test
-  //    final CustomObjectQuery query = customObjectSyncer.getQuery();
-  //
-  //    // assertion
-  //    final List<String> excludedContainerNames = getExcludedContainerNames(runnerName);
-  //    assertThat(query)
-  //        .isEqualTo(
-  //            CustomObjectQuery.ofJsonNode()
-  //                .plusPredicates(
-  //                    customObjectQueryModel ->
-  //                        customObjectQueryModel.container().isNotIn(excludedContainerNames)));
-  //  }
-  //
```

Comment:
The migration of this test is missing.

Reply:
I removed it because it is already tested in `of_ShouldCreateCustomObjectSyncerInstance`. So this test was redundant.